kong/plugins/rate-limiting/handler.lua

```diff
@@ -29,10 +29,12 @@ local function get_identifier(conf)
     end
   elseif conf.limit_by == "credential" then
     identifier = ngx.ctx.authenticated_credential and ngx.ctx.authenticated_credential.id
+  elseif conf.limit_by == "ip" then
+    identifier = ngx.var.remote_addr
   end
 
   if not identifier then
-    identifier = ngx.var.remote_addr
+    identifier = ngx.var.server_addr .. "-" .. ngx.var.server_name .. ngx.var.pid .. "-" .."-global"
```

Comment:
Is this really a per worker ratelimit as there is `ngx.var.pid` (provided that no worker in cluster ever uses a same process id, which you have no gurantee)? Or do I read this incorrectly?

Reply:
Hi,
Thank you for the correction, my will is to have a limit per "kong" instance. It makes sense to simplify the identifier like that as long as this value is never empty or valuated at 127.0.0.1 ?
I am not familiar with nginx / kong internals.